modules/fbx/fbx_parser/FBXImportSettings.h

```diff
@@ -80,60 +80,52 @@ namespace FBXDocParser {
 
 /** FBX import settings, parts of which are publicly accessible via their corresponding AI_CONFIG constants */
 struct ImportSettings {
-	ImportSettings() :
-			strictMode(true), readAllLayers(true), readAllMaterials(true), readMaterials(true), readTextures(true), readCameras(true), readLights(true), readAnimations(true), readWeights(true), preservePivots(true), optimizeEmptyAnimationCurves(true), useLegacyEmbeddedTextureNaming(false), removeEmptyBones(true), convertToMeters(false) {
-		// empty
-	}
-
 	/** enable strict mode:
      *   - only accept fbx 2012, 2013 files
      *   - on the slightest error, give up.
      *
      *  Basically, strict mode means that the fbx file will actually
-     *  be validated. Strict mode is off by default. */
```

Comment:
@RevoluPowered I removed mentions of default values since they're better documented by the code itself (and this one was wrong).